dapp-manager/src/main/kotlin/jp/co/soramitsu/dapp/manager/registration/DappAccountRegistrator.kt

```diff
@@ -0,0 +1,178 @@
+                            domain,
+                            Utils.toHex(keyPair.public.encoded)
+                        ),
+                        IrohaCommand.CommandSetAccountDetail(
+                            dappAccountsStorage,
+                            "$name$domain",
+                            domain
+                        )
+                    )
+                ),
+                // Second step is to give permissions from the user to brvs and registration service
+                // Here we need our own key to sign this stuff
+                IrohaTransaction(
+                    newUserAccountId,
+                    ModelUtil.getCurrentTime(),
+                    listOf(
+                        IrohaCommand.CommandGrantPermission(
+                            brvsAccountId,
+                            Primitive.GrantablePermission.can_set_my_quorum_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            brvsAccountId,
+                            Primitive.GrantablePermission.can_add_my_signatory_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            brvsAccountId,
+                            Primitive.GrantablePermission.can_remove_my_signatory_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            irohaConsumer.creator,
+                            Primitive.GrantablePermission.can_set_my_quorum_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            irohaConsumer.creator,
+                            Primitive.GrantablePermission.can_add_my_signatory_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            irohaConsumer.creator,
+                            Primitive.GrantablePermission.can_remove_my_signatory_VALUE
+                        )
+                    )
+                ),
+                // Also we need to give permissions from requests account perspective
+                IrohaTransaction(
+                    requestAccountId,
+                    ModelUtil.getCurrentTime(),
+                    listOf(
+                        IrohaCommand.CommandGrantPermission(
+                            newUserAccountId,
+                            Primitive.GrantablePermission.can_set_my_account_detail_VALUE
+                        )
+                    )
+                ),
+                // Finally we need to add user's original pub key to the signatories list
+                // But we need to increase user's quorum to prohibit transactions using only user's key
+                // Several moments later BRVS react on the createAccountTransaction and
+                // user's quorum will be set as 1+2/3 of BRVS instances for now
```

Comment:
BTVS mentioning instead of Dapps. May confuse newby

Reply:
It confused you since mentioning BRVS is correct here. dApps instances are going to play a role of 'clients' in 'dapp' domain.